src/plugins/intel_gpu/include/intel_gpu/runtime/format.hpp

```diff
@@ -382,6 +382,7 @@ struct format {
     constexpr operator type() const { return value; }
 
     std::string to_string() const;
+    format get_default_format() const;
```

Comment:
why is this needed?

Reply:
it is a bit weird because it is calling "Default format" from a format, though the caller itself is a format.